### ISSUE TITLE
Change cpuclock plugin to give full options in arm11 kernel/simplify.

### ISSUE THREAD

**Issue by 133794m3r** [44670/BootNTR#45]
Currently the options given in cpuplug(built in or otherwise) doesn't include all of the options given by the arm 11 kernel, all it has is 1x, and 3x but the hardware allows for 1x(o3ds mode), 2x, and 3x(n3ds mode).

If either the plug or ntr was open source I'd try my hand at modifying the cpuclock feature and verifying that it works just but fine but since it's not his is my only option.

I would also like to see the cpuclock options changed. Currently there are way too many options. There is no reason for the option of disabling the l2 cache when running at max clock. I would lke to see the options changed to the following also.

268mhz no l2(default)
268mhz l2 enabled good
536mhz l2 enabled better
804mhz l2 enabled best

I have testd games with insane loading times like lego batman 2 and it's loading times go from 18-20s with defaults but then with l2 enabled it goes to ~15s and then with max speed+804mhz it drops to 12s. And that's just not enough of a loading time decrease for me to want to drain the battery even more than it already decreases by. 2x should give a nice middle between the defaults(slow as hell) and max for games that run bad at the defaults but would probably run at least somewhat well at 3x, plus the battery life decrease should be minimal at least compared with going full force.

I have seen people on youtube and compared with my owntests for default loading speeds, both with flash cart, cia(class 10), and game cart and they all load at a very similar rate. All of the lego gaems load like crap especially lego city undercover and this minor change should greatly help lots of people such as myself who want better loading speeds but don't want to have the overall batterylife decrease by even more.

TL;DR
Modify cpuplug to allow setting the register to 2x cpuclock, and reduce options by only having default(no l2) but the rest of the cpu options with the l2 cache enabled to reduce options shown.

**Comment by 133794m3r** [comment]
no comment just closing it? that's odd.

**Comment by 133794m3r** [comment]
no comment just closing it? that's odd.

**Comment by 133794m3r** [comment]
no comment just closing it? that's odd.

**Comment by 133794m3r** [comment]
no comment just closing it? that's odd.

**Comment by 133794m3r** [comment]
no comment just closing it? that's odd.

**Comment by 44670** [comment]
currently no plans to do that, thank for suggestions anyway.
